util/logging.go

```diff
@@ -112,6 +113,7 @@ func TogglePanicEmptyUserAgent(willPanic bool) {
 func newLogger(logpath string) *log.Logger {
 	var err error
 	var file *os.File
+	// #nosec G302 G304 -- The file has read permissions for groups as it is just a log
 	if OverwriteLog {
 		file, err = os.OpenFile(logpath, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0640)
```

Comment:
Possibly adding `filepath.Clean` here?
Also, what problem would we have if we restrict permissions to `0600`?